R/parameters.R

```diff
@@ -0,0 +1,210 @@
+#' Read in disease process parameters from an external file or files
+#'
+#' @param generation_interval_path,delay_interval_path,right_truncation_path
+#'   Path to a local file with the parameter PMF. See [read_interval_pmf] for
+#'   details on the file schema. The parameters can be in the same file or a
+#'   different file.
+#' @param disease One of `COVID-19` or `Influenza`
+#' @param as_of_date Use the parameters that were used in production on this
+#'   date. Set for the current date for the most up-to-to date version of the
+#'   parameters and set to an earlier date to use parameters from an earlier
+#'   time period.
+#' @param state Used only for parameters with a state-level estimate (i.e., only
+#'   right-truncation). The two-letter uppercase state abbreviation.
+#'
+#' @return A named list with three PMFs. The list elements are named
+#'   `generation_interval`, `delay_interval`, and `right_truncation`. If a path
+#'   to a local file is not provided (NA or NULL), the corresponding parameter
+#'   estimate will be NA in the returned list.
```

Comment:
```suggestion
#'   estimate will be NA in the returned list.
#'  @details `generation_interval_path` is required because the generation interval is 
#'   a required parameter for $R_t$ estimation. `delay_interval_path` and 
#'  `right_truncation_path` are optional (but strongly suggested).
```

Reply:
I think it's worth documenting this logic somewhere other than the PR notes.